airbyte-ci/connectors/base_images/base_images/python/sanity_checks.py

```diff
@@ -70,14 +70,10 @@ async def check_python_image_has_expected_env_vars(python_image_container: dagge
     """
     expected_env_vars = {
         "PYTHON_VERSION",
-        "PYTHON_PIP_VERSION",
```

Comment:
I have no clue where they are set, and they don't seem to be used anywhere. Code for the sake of code must die.